### PR TITLE
initialize file

### PR DESCRIPTION
**Pull request by BBozhikov**

### CODE REVIEW

**Comment by valentin-aleksandrov** [comment]
I see that you're missing a .gitignore file.

**Comment by BBozhikov** [comment]
what is gitignore

File: main.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+    else:
+        start_count_down()
+
+
+# ---------------------------- UI SETUP ------------------------------- #
+
+window = Tk()
+window.title("Pomodoro Timer")
+window.config(bg=YELLOW, padx=100, pady=50)
+window.tk.call('tk', 'scaling', 1.4)
```
Comment (valentin-aleksandrov) [inline main.py:74]: As we've discussed this is the standard scaling:
```suggestion
window.tk.call('tk', 'scaling', 1)
```

**Comment by valentin-aleksandrov** [comment]
Please, specify in the `readme.md` file the version of python that I need in order to run this project.

**Pull request merged**